specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-04-25-preview/examples/Machines_CreateOrUpdate.json

```diff
@@ -0,0 +1,87 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscriptionId}",
+    "resourceGroupName": "myResourceGroup",
+    "machineName": "myMachine",
+    "api-version": "2023-04-25-preview",
+    "parameters": {
+      "location": "eastus2euap",
+      "properties": {
+        "vmId": "b7a098cc-b0b8-46e8-a205-62f301a62a8f",
+        "locationData": {
+          "name": "Redmond"
+        },
+        "clientPublicKey": "string",
+        "parentClusterResourceId": "{AzureStackHCIResourceId}",
+        "privateLinkScopeResourceId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.HybridCompute/privateLinkScopes/privateLinkScopeName"
+      },
+      "identity": {
+        "type": "SystemAssigned"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.HybridCompute/machines/myMachine",
+        "name": "myMachine",
+        "location": "eastus2euap",
+        "tags": null,
+        "identity": {
+          "type": "SystemAssigned",
+          "principalId": "string",
+          "tenantId": "string"
+        },
+        "type": "Microsoft.HybridCompute/machines",
+        "kind": null,
+        "properties": {
+          "provisioningState": "Succeeded",
+          "agentVersion": null,
+          "status": null,
+          "lastStatusChange": null,
+          "errorDetails": null,
+          "displayName": null,
+          "machineFqdn": null,
+          "vmUuid": null,
+          "osSku": null,
+          "domainName": null,
+          "adFqdn": null,
+          "dnsFqdn": null,
+          "osVersion": null,
+          "osType": null,
+          "osProfile": {
+            "computerName": null,
+            "windowsConfiguration": {
+              "patchSettings": {
+                "assessmentMode": null,
+                "patchMode": null
+              }
+            },
+            "linuxConfiguration": {
+              "patchSettings": {
+                "assessmentMode": null,
+                "patchMode": null
+              }
+            }
+          },
```

Comment:
Are there supposed to be this many nulls?

Reply:
Yes, see previous comment